draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Verifying Firmware Provenance for large and complex systems
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by checking an accompanying signature or by creating a hash of measurements and comparing them to a reference value.
```

Comment:
```suggestion
Authentication takes place, for example, by verifying a signature or by creating a hash of measurements and comparing them to a reference value.
```

Reply:
hm. verification is too overloaded for my taste. If "checking" is not good enough... maybe "validating" instead?